subsys/bluetooth/host/hci_core.c

```diff
@@ -2128,6 +2183,25 @@ static void enh_conn_complete(struct bt_hci_evt_le_enh_conn_complete *evt)
 
 	bt_conn_set_state(conn, BT_CONN_CONNECTED);
 
+#if defined(CONFIG_BT_USER_PHY_UPDATE)
+	if (IS_ENABLED(CONFIG_BT_EXT_ADV) &&
+	    BT_FEAT_LE_EXT_ADV(bt_dev.le.features)) {
+		u8_t tx_phy, rx_phy;
+		int err;
+
+		err = hci_le_read_phy(conn, &tx_phy, &rx_phy);
+		if (err) {
+			BT_WARN("Failed to read PHY (%d)", err);
+		} else if (!(tx_phy == conn->le.phy.tx_phy &&
+			     rx_phy == conn->le.phy.rx_phy)) {
+			conn->le.phy.tx_phy = tx_phy;
+			conn->le.phy.rx_phy = rx_phy;
+
+			notify_le_phy_updated(conn);
+		}
```

Comment:
When a new connection is established on LE Coded PHY, the initial Data Length Time parameters are initialized to 2704. See Core_v5.2, Vol 6, Section 4.5.10. 
Should we inform the application about this?

Reply:
I'm unsure about this one. but I guess it would make sense for consistency. Anybody else has an opinion on this?

Reply:
I think this is better suited for documentation than actually calling the callback here. I just think it's wasted time and resources in both code footprint and application handling.

Reply:
Moved this before setting state to `BT_CONN_CONNECTED`, which would call the connected callback. So now the application can call bt_conn_get_info in `connected` and the PHY would be correct. The callback is not called anymore.